rest-service/manager_rest/configure_manager.py

```diff
@@ -509,8 +518,69 @@ def _add_deployments_filter(sys_filter_dict, creator, tenant, now):
     db.session.add(models.DeploymentsFilter(**sys_filter_dict))
 
 
+def _generate_db_config_entries(cfg):
+    manager_private_ip = cfg['manager'].get('private_ip', 'localhost')
+    default_file_server_url = f'https://{ipv6_url_compat(manager_private_ip)}'\
+                              f':{DEFAULT_INTERNAL_REST_PORT}/resources'
+
+    prometheus_cfg = cfg.get('prometheus', {})
+    rest_cfg = {
+        'rest_service_log_path': REST_LOG_DIR + '/cloudify-rest-service.log',
```

Comment:
```suggestion
        'rest_service_log_path': os.path.join(REST_LOG_DIR, 'cloudify-rest-service.log'),
```
(+flake, this line is too long i think)
You already use os.path.join up top, so let's maybe stay consistent